src/settings/Models/ReminderFeesPolicy/ReminderFeesPolicy.test.js

```diff
@@ -23,12 +23,26 @@ describe('ReminderFeesPolicy', () => {
 
   it('"defaultReminderFeesPolicy" should return correct data', () => {
     const expectedPolicy = {
-      countClosed: true,
-      ignoreGracePeriodRecall: true,
-      clearPatronBlockWhenPaid: true,
+      countClosed: false,
```

Comment:
Is it deliberate that the values provided for some of the existing field have changed?

Reply:
Yes @MikeTaylor, the previous implementation was buggy. What is coming by default from the server is `false` so we just need to make sure we align with that here.